packages/server/modules/workspaces/domain/operations.ts

```diff
@@ -48,6 +48,12 @@ export type GetUserDiscoverableWorkspaces = (args: {
   userId: string
 }) => Promise<LimitedWorkspace[]>
 
+export type EligibleWorkspace = LimitedWorkspace & { role?: WorkspaceRoles }[]
```

Comment:
What's the roles array for? It's not immediately obvious, so a comment might be nice